include/oc_pki.h

```diff
@@ -200,7 +188,46 @@ int oc_pki_add_trust_anchor(size_t device, const unsigned char *cert,
 void oc_pki_set_security_profile(size_t device,
                                  oc_sp_types_t supported_profiles,
                                  oc_sp_types_t current_profile, int mfg_credid);
+
+/**
+ * @brief TLS peer connection
+ *
+ */
+struct oc_tls_peer_t;
+
+/**
+ * @brief Callback invoked after ocf verifies the certificate chain. For each
+ * certificate in the chain, the callback is invoked with the depth of the
+ * certificate in the chain.
+ *
+ * @param peer TLS peer connection
+ * @param crt Certificate
+ * @param depth Depth of the certificate chain, 0 is the leaf certificate.
+ * @param flags Verification flags from mbedtls_x509_crt_verify(), see
+ * https://github.com/Mbed-TLS/mbedtls/blob/10ada3501975e7abab25a7fa28e9e8e0f6b4259f/include/mbedtls/x509.h#L99
+ *
+ * @return true if the cloud manager should continue sending keepalive pings,
```

Comment:
The documentation doesn't seem to match the signature -> documentation says the return type is `bool` but the signature says it is `int`.

Reply:
fixed